product-service/src/handlers/createProduct.ts

```diff
@@ -0,0 +1,28 @@
+import { APIGatewayProxyEvent, Context } from 'aws-lambda';
+import 'source-map-support/register';
+import productService from '../services/productService';
+import { Headers } from '../utils/constants';
+import { Product } from '../utils/interfaces';
+
+export const handle = async (event: APIGatewayProxyEvent, _context: Context) => {  
+    try {    
+        console.log(event);
+        const body = JSON.parse(event.body) as Product;  
+        body.price = Math.round(body.price * 100);
+        await productService.createProduct(body);
+        return {
+            statusCode: 204,
+            body: JSON.stringify({message: "Nor content!"}, null, 2),
```

Comment:
If any error happens in `productService.createProduct`, won't it go straight to `catch` block? So that `Nor content!` error can't get